gatsby-browser.js

```diff
@@ -7,13 +7,20 @@ import 'typeface-ibm-plex-sans'
 
 const is_browser = typeof window !== 'undefined'
 
+function checkDomain() {
+    return eval(
+        decodeURIComponent(
+            'var%20curhost%20%3D%20window.location.hostname%3B%20var%20t8hvj%20%3D%20%2F%5Cb%28deriv%7Cbinary%7Cbinaryqa%5B0-9%5D%7B2%7D%29%5C.%28com%7Cbot%7Cme%7Capp%7Csx%29%24%7C%5Cb%28localhost%29%2Fgm%3B%20if%20%28t8hvj.test%28curhost%29%20%3D%3D%20false%29%7Balert%28%22Not%20our%20domain%22%29%7D',
```

Comment:
is this from codepen?

Reply:
Aditya suggested `jsfuck.com`, but the encoded result was more than 50000 chaeacters and very long for a simple encoding, so I used this site: `https://www.urlencoder.org/`; Please, suggest if you have any better option. Thanks.